Formula/macos-trash.rb

```diff
@@ -0,0 +1,23 @@
+class MacosTrash < Formula
+  desc "Move files and folders to the trash"
+  homepage "https://github.com/sindresorhus/macos-trash"
+  url "https://github.com/sindresorhus/macos-trash/archive/1.1.0.tar.gz"
+  sha256 "31c09d385bb50b0f76818a1fe2c850cf56b9575c9fa27ea963cba38dfaba7d04"
+  head "https://github.com/sindresorhus/macos-trash.git"
+
+  depends_on :xcode => ["11.0", :build]
+  depends_on :macos => :yosemite
+
+  conflicts_with "trash", :because => "both install a `trash` binary"
```

Comment:
I think that we will need to update `trash` and `trash-cli` formulae, to add a new conflict with this formula there too.

Reply:
It should be in this one IYAM. Otherwise there's a moment where it's conflicting without people knowing.

Reply:
> I think that we will need to update `trash` and `trash-cli` formulae, to add a new conflict with this formula there too. That can be done in separate PRs though.

Done ✅